typings/artifacts.d.ts

```diff
@@ -53,6 +53,8 @@ declare global {
       CrawlableLinks: {href: string, text: string}[];
       /** CSS coverage information for styles used by page's final state. */
       CSSUsage: {rules: Crdp.CSS.RuleUsage[], stylesheets: Artifacts.CSSStyleSheetInfo[]};
+      /** Information on the document's Doctype, specifically the name, publicId, and systemId */
```

Comment:
maybe also something like "Properties will be the empty string if not present on the doctype." or something? I was expecting `undefined` or something but at least the latter two are specced to be the empty string if not present